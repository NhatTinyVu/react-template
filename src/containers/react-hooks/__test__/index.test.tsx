import React from 'react';
import { shallow } from 'enzyme';
import toJSON from 'enzyme-to-json';
import ReactHooks from '..';

describe('ReactHooks', () => {
  test('renders correctly', (): void => {
    const wrapper = shallow(<ReactHooks />);
    expect(toJSON(wrapper)).toMatchSnapshot();
  });
});
