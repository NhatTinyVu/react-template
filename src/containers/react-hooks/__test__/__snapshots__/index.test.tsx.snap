// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ReactHooks renders correctly 1`] = `
<Adapter
  className="content-center"
>
  <h3
    className="text-center"
  >
    React Hooks Templates
  </h3>
  <p>
    Please select the template
  </p>
</Adapter>
`;
